src/main/java/duke/file_management/CreateFile.java

```diff
@@ -0,0 +1,50 @@
+package duke.file_management;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * To create a file in the user's local device.
+ *
+ * @author Justin Ng Jie Ern
+ */
+public class CreateFile {
```

Comment:
Would using a noun for naming class be better?

Reply:
As there is a Java API called FileWriter, I had to name my file writer class WriteFile. As i wanted some consistency between this 2 classes, I named this class CreateFile. 

Would welcome your suggestions!